cluster_configs/config.yaml

```diff
@@ -0,0 +1,25 @@
+tft:
+  - name: "Test 1"
+    namespace: "default"
+    # test cases can be specified individually i.e "1,2,3,6" or as a range i.e. "1-9,15-19"
+    test_cases: "1"
+    duration: "30"
+    # Location of artifacts from run can be specified: default <working-dir>/ft-logs/
+    # logs: "/tmp/ft-logs"
+    connections:
+      - name: "Connection_1"
+        # supported types: iperf-tcp, iperf-udp, netperf-tcp-stream, netperf-tcp-rr
+        type: "iperf-tcp"
+        instances: 1
+        server:
+          - name: "$worker"
```

Comment:
@wizhaoredhat can you PTAL at this line?

Reply:
lgtm for now, but we want tft to be able to have something like "{{worker(0)}}" in place that will query the cluster with `oc`